src/opers.c

```diff
@@ -4895,27 +4895,28 @@ Obj DoVerboseMutableAttribute (
 #define ImpliedWriteGuard(x)
 #endif
 
-#define WRAP_NAME(fname, name, addon) \
-    do { \
-        UInt name_len = GET_LEN_STRING(name); \
-        UInt addon_len = sizeof(addon) - 1; \
-        char *ptr; \
-        fname = NEW_STRING( name_len + addon_len + 2 ); \
-        ImpliedWriteGuard(fname); \
-        ptr = CSTR_STRING(fname); \
-        memcpy( ptr, addon, addon_len ); ptr += addon_len; \
-        *ptr++ = '('; \
-        memcpy( ptr, CSTR_STRING(name), name_len ); ptr += name_len; \
-        *ptr++ = ')'; \
-        *ptr = 0; \
-        RetypeBag( fname, IMMUTABLE_TNUM(TNUM_OBJ(fname)) ); \
-    } while(0)
+Obj WRAP_NAME(Obj name, const char *addon)
```

Comment:
If the old macro was not visible outside this file, should the new function not be static?